src/store/reducers/likeDislikeCommentReducer/likeDislikeCommentReducer.test.js

```diff
@@ -0,0 +1,90 @@
+// action types
+import {
+  LIKE_COMMENT_SUCCESS,
+  LIKE_COMMENT_FAILURE,
+  DISLIKE_COMMENT_SUCCESS,
+  DISLIKE_COMMENT_FAILURE,
+  LIKE_COMMENT_STATE,
+} from 'store/actions/likeDisLikeCommentTypes';
+
+// reducer
+import likeDislikeCommentReducer from '.';
+
+/**
+ * This is a function which tests the
+ * likeDislikeCommentReducer
+ */
```

Comment:
This block comment in unnecessary, make sure the describe block gives enough context of what is happening with the test suit